src/app/authorization/login/login.component.css

```diff
@@ -1,13 +1,17 @@
-*{
-    font-family: Roboto;
-}
-h6{
-    margin-top: 14px;
-}
-.card{
-    margin: 0 auto; /* Added */
-    /* float: none; Added */
-    /* margin-bottom: 30px; Added */
-    border-radius: 15px;
-    height: auto;    
-}
\ No newline at end of file
+* {
```

Comment:
why do we need to apply roboto font to all css ? is there any requirement

Reply:
yes need same font in whole project so need it but not in all css file so i changed it and give the font to the style.css file.